### ISSUE TITLE
v1.0.0: Integrate multi-fidelity lidar sampling via transfer learning

### ISSUE THREAD

**Issue by zcrennenLANL** [lanl/CHM-MS-net-Canopy-Height-Model#3]
The code is set up to accept custom lidar, but generally requires at least 1024 256x256m tiles, or a spatial area of ~67 km^2, approximately an 8km x 8km area (5 miles x 5 miles). While this spatial coverage may be common for aerially collected lidar, this is difficult to cover with UAS lidar. 

To leverage UAS lidar and mitigate its issue of small spatial coverage, we can employ transfer learning. Pretrain a model on at least 1024-ish tiles of the most appropriate 3DEP lidar, then freeze the weights of all but the last 2-ish layers and fine tune with high quality UAS lidar.

Note: This will be a relatively significant effort and is nontrivial. As UAS lidar will need to be split train/val/test, this will result in even less data available for training. It may still make sense to just use UAS lidar for test as the test set in this case will be very small. Additionally, a temporal gap will likely exist between the 3DEP collection and UAS collection.
